suite-native/atoms/src/TransactionItem/TransactionItem.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Icon } from '@trezor/icons';
+import { capitalizeFirstLetter } from '@trezor/utils';
+
+import { Box } from '../Box';
+import { Text } from '../Text';
+
+type TransactionStatus = 'received' | 'sent';
+
+type TransactionItemProps = {
+    cryptoCurrency: string;
```

Comment:
not sure about name of this prop, I would go with only `currencyTitle` or please check what they are using in Suite

Reply:
https://github.com/trezor/trezor-suite/pull/5783/commits/eebfdd500df1a8017caed50b05949391633cbb86